### ISSUE TITLE
Issue when using in typescript project

### ISSUE THREAD

**Issue by jondrummond-codeo** [rse/pure-uuid#22]
The latest version (1.8.0) throws the following error when used in a typescript project. The previous version (1.7.0) works fine.

```
node_modules/pure-uuid/uuid.d.ts:70:1 - error TS2309: An export assignment cannot be used in a module with other exported elements.

70 export = UUID;
   ~~~~~~~~~~~~~~
```

**Comment by rse** [comment]
For me, under TypeScript 5.2.2 I did not receive the problem. But I've tried to fix it in pure-uuid version 1.8.1. Can you retry with this version now?

**Comment by jondrummond-codeo** [comment]
@rse That seems to have sorted the issue out. Thanks.
